ecosystem/sep-0008.md

```diff
@@ -305,6 +306,8 @@ There are two possible values for `action_method`.
    }
    ```
 
+When no further actions can be taken by the user, the client can re-submit the transaction to the approval server. However, there is no guarantee to receive a `"success"` status in the following call. Clients should expect to receive a response with any of the status.
```

Comment:
This statement looks like it relates only to the POST case, and should be indented with the POST details.

I also think you should move this paragraph up to line 267 and put it before the parameter lists and examples, because that's where the discussion about how to use it is and this is part of that.

```suggestion
  When no further actions can be taken by the user, the client can re-submit the transaction to the approval server. However, there is no guarantee to receive a `"success"` status in the following call. Clients should expect to receive a response with any of the status.
```

Reply:
It's actually meant to be used by both GET and POST. That's why I didn't have the indentation there.

I thought adding it after `Following the Action URL` makes more sense, but it's indeed not so clear which case it applies. Any suggestion?

Reply:
How about another sub-title `After Completing the Action(s) Required`?

Reply:
Ah gotcha. Can you move it to line 255? I think it belongs there. 
```suggestion
When no further action is to be taken by the user, the client can re-submit the transaction to the approval server. However, there is no guarantee to receive a `"success"` status in the following call. Clients should expect to receive a response with any of the status.
```

Reply:
https://github.com/stellar/stellar-protocol/pull/858/commits/174d7fb5b7c735a807303c7b916f9fb25479d0c6